src/test/java/cloud/fogbow/accs/core/processors/SyncProcessorTest.java

```diff
@@ -0,0 +1,301 @@
+package cloud.fogbow.accs.core.processors;
+
+import cloud.fogbow.accs.core.datastore.DatabaseManager;
+import cloud.fogbow.accs.core.datastore.orderstorage.AuditableOrderIdRecorder;
+import cloud.fogbow.accs.core.datastore.orderstorage.AuditableOrderStateChange;
+import cloud.fogbow.accs.core.models.Record;
+import cloud.fogbow.accs.core.models.orders.Order;
+import cloud.fogbow.accs.core.models.orders.OrderState;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+import org.mockito.Mockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.modules.junit4.PowerMockRunnerDelegate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+@PowerMockIgnore({"javax.management.*"})
+@PowerMockRunnerDelegate(SpringRunner.class)
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(SyncProcessor.class)
+@SpringBootTest
```

Comment:
Try migrating these test setup annotations to the BaseUnitTests class and comment on why using line 28 to running the tests.